### PR TITLE
Display server side error messages

### PR DESCRIPTION
**Pull request by trizin**
image

### CODE REVIEW

File: frontend/src/pages/SubmitProposal.svelte [hunk 1]
```diff
@@ -312,6 +336,9 @@ Community Value — How does the project add value to the overall Ocean Communit
               />
             {/each}
           </div>
+          {#if errortext}
+            <div class="text-red-500">{errortext}</div>
```
Comment (idiom-bytes) [inline frontend/src/pages/SubmitProposal.svelte:340]: Thank you for adding this to SubmitProposal!

File: frontend/src/pages/CreateProject.svelte [hunk 2]
```diff
@@ -252,13 +252,18 @@ Co-founder at xxx`,
...
           } else if (res.status === 400) {
             console.log("Couldn't update project: ", res);
-            errorMessage =
-              "Error updating project. Please check fields. " + res;
+            res = await res.text();
+            try {
+              res = JSON.parse(res);
+            } catch (e) {}
+            errorMessage = `Error creating project. Please check fields. ${
+              res.message ?? res
```
Comment (idiom-bytes) [inline frontend/src/pages/CreateProject.svelte:265]: Great to see the error being visualized.

`res.message ?? res`
Am I reading this right? is this ternary?

in other words...
`if res.message !== undefined then return res else return null;`

**Review by idiom-bytes — COMMENTED** [review]
Looks great! Great to see the underlying error being reported.

File: frontend/src/pages/CreateProject.svelte (hunk 2 above)
Comment (trizin) [reply frontend/src/pages/CreateProject.svelte:265]: This is [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)

**Pull request merged**